### ISSUE TITLE
Support attention for Vimeo auto contribution

### ISSUE THREAD

**Issue by mandar-brave** [brave/brave-browser#4017]
Support capturing attention for Vimeo as a platform. 
Replicate YT functionality; including support for unverified Vimeo publishers to be treated the same way as unverified publishers elsewhere on other channels.

In phase 1 we will add support for auto-contribution.
In phase 2 we will add panel/tip support https://github.com/brave/brave-browser/issues/4391

cc @NejcZdovc

**Comment by NejcZdovc** [comment]
in first iteration we will support only AC functionality. @mandar-brave please create separate issue for panel/tipping support

**Comment by NejcZdovc** [comment]
blocked on https://github.com/brave/brave-browser/issues/4005

**Comment by GeetaSarvadnya** [comment]
Verification passed on


Brave | 0.68.104 Chromium: 75.0.3770.100 (Official Build) dev (64-bit)
-- | --
Revision | cd0b15c8b6a4e70c44e27f35c37a4029bad3e3b0-refs/branch-heads/3770@{#1033}
OS | Windows 10 OS Version 1803 (Build 17134.523)

- Verified the test plan from https://github.com/brave/brave-core/pull/2247
- Logged https://github.com/brave/brave-browser/issues/5295

![image](https://user-images.githubusercontent.com/38657976/61538777-bfe76380-aa57-11e9-8c9a-dfcb6bc8bbf3.png)
